packages/server/src/routes/modlists.ts

```diff
@@ -792,6 +795,13 @@ search
     if (validated.cursor) {
       conditions.push(lt(modListUser.id, validated.cursor))
     }
+    if (validated.query) {
```

Comment:
Please add the corresponding unit tests in packages/server/test/modlists.test.ts.